tests/test_transform.py

```diff
@@ -180,10 +180,94 @@ def checkTransform(self, fromName, toName):
                 with self.assertRaises(InvalidParameterError, msg=msg):
                     transformClass(badPolyMap)
 
+    def checkGetInverse(self, fromName, toName):
+        """Test Transform<fromName>To<toName>.getInverse
+
+        Parameters
+        ----------
+        fromName, toName : string
+            the prefixes of the transform's endpoints (e.g., "Point2" for a
+            Point2Endpoint)
+        """
+        transformClassName = "Transform{}To{}".format(fromName, toName)
+        transformClass = getattr(afwGeom, transformClassName)
+        baseMsg = "transformClass={}".format(transformClass.__name__)
+        for nIn in self.goodNaxes[fromName]:
+            for nOut in self.goodNaxes[toName]:
+                # Transform with both forward and inverse transforms
+                self.checkGetInverseWithMap(
+                    transformClass(makePolyMap(nIn, nOut)),
+                    nIn,
+                    "{}, Map={}, nIn={}, nOut={}".format(
+                        baseMsg, "Invertible", nIn, nOut))
+                # TODO: add more test cases once DM-9826 resolved
+
+    def checkGetInverseWithMap(self, transform, nIn, msg):
+        """Test Transform<fromName>To<toName>.getInverse with a specific mapping.
```

Comment:
If I am reading the code correctly:
- `checkGetInverse` creates all possible transforms for a given pair of endpoint names, creating each frame set or mapping as it sees fit.
- `checkGetInverseWithMap` checks one transform.

I'm asking for a less confusing name scheme. To be fair I'll propose one, but do as you like:
- Rename `checkGetInverseWithMap` to `checkGetInverse` and ditch old version of the latter.
- Make `checkTransform` call `checkGetInverseWithMap`

Or if that's too much work for `checkTransform` you could make a new method `testGetInverse` and have it build all valid transforms that you want to test, then call `checkTransform` from there.

I wonder if I should break `checkTransform` up a bit, e.g. I could imagine a function by that name which takes a valid transform as an argument and checks basics. Then `testTransform` would have to construct the transforms. I'm not sure where I would move the code that tries to construct invalid transforms -- perhaps a new `testX` method. Comments?